packages/samples/sources/move-basics/assert-and-abort.move

```diff
@@ -18,6 +18,9 @@ if (!user_has_access) {
 if (user_has_access) {
    abort(1)
 };
+
```

Comment:
I'm not sure why these changes (and the other changes in the move code) are here